docs/parallelism.md

```diff
@@ -0,0 +1,31 @@
+# Limiting parallelism
+
+You can restrict the number of simultaneous workflow executions.
+
+## Controller level
+
+You can limit the total number of workflows that can execute at any one time in the [workflow controller ConfigMap](./workflow-controller-configmap.yaml).
+
+```yaml
+data:
+  parallelism: "10"
+```
+
+You can also limit the number of workflows that can execute in a single namespace.
+
+```yaml
+data:
+  namespaceParallelism: "4"
+```
+
+Workflows that are executing but restricted from running more nodes due to the other mechanisms will still count towards the parallelism limits.
+
+### Priority
+
+Workflows can have a `priority` set in their specification.
+
+Workflows with a higher priority number that have not started due to controller level parallelism will be started before lower priority workflows.
```

Comment:
```suggestion
Workflows that have not started yet will be queued: Workflows with higher priority numbers will start before lower priority ones.
```

simplify.

although this feels identical to the "Queuing" section in the "Synchornization" page as I mentioned above

Reply:
This is a different queue, unrelated to any synchronization based queues.

You need to pass through this queue before you can enter any synchronization based queue, and this is important to understand.

In the Synchronization section the queue is about "What happens when a mutex isn't available". At the moment you get no option, you enter a queue. We may in future have alternative options to skip this section of the workflow, or wait in the queue for some limited time.

It's probable that people can guess they would enter a queue, and here the details they cannot guess at are the ordering of the queue. It's not a FIFO as you may guess as a simple implementation, it's priority then workflow creation time.

Reply:
Your rewrite here doesn't explain why workflows may not be started, so I've adopted some of your rewrite.